### PR TITLE
Batch support + Private Cloud + Testing

### PR DESCRIPTION
**Pull request by madisonmay**
Should be ready for code review after some final tweaks tomorrow morning.

### CODE REVIEW

**Comment by madisonmay** [comment]
Some tests should be skipped if batch API credentials aren't available.

**Comment by madisonmay** [comment]
TODO:

- [ ] Skip tests if credentials aren't available
- [ ] Fix failing codeship tests
- [ ] Update version numbers
- [ ] Merge PR 
- [ ] Push to npm

File: CHANGES.txt [hunk 1]
```diff
@@ -6,3 +6,4 @@ v0.1.7, Fri Nov 7 -- Updated to include new texttags API
 v0.1.9, Fri Nov 7 -- Readme update
 v0.1.10, Wed Nov 26 -- Batch support
 v0.1.11, Thu Dec 19 -- Batch authorization 
+v0.2.0, Wed March 18 -- Private cloud support and auth configuration
```
Comment (anneCarlson) [inline CHANGES.txt]: Worth mentioning a README update as well.

**Comment by anneCarlson** [comment]
This looked fine to me, however, I think we should make a habit of submitting or code review more often so we get smaller reviews to look over. It's really easy to glaze over with a review this long.

**Comment by madisonmay** [comment]
:+1: ^

**Pull request merged**